Assets/_BForBoss/_Character/Scripts/PlayerWallRunBehaviour.cs

```diff
@@ -322,6 +402,34 @@ private static bool GetSmallestRaycastHitIfValid(RaycastHit[] array, out Raycast
             }
             return validRaycast;
         }
+        
+        private bool ProcessRaycasts(Vector3[] directions, out RaycastHit smallestRaycastResult, Transform origin, int layerMask, float maxDistance)
+        {
+            RaycastHit[] hits = new RaycastHit[directions.Length];
+            for (int i = 0; i < directions.Length; i++)
+            {
+                Vector3 localDirection = origin.TransformDirection(directions[i]);
+                Physics.Raycast(origin.position, localDirection, out hits[i], maxDistance, layerMask);
+                if (hits[i].collider != null)
+                {
+                    Debug.DrawRay(origin.position, localDirection * hits[i].distance, Color.green);
+                }
+                else
+                {
+                    Debug.DrawRay(origin.position, localDirection * maxDistance, Color.red);
```

Comment:
Could wrap this with the `#if Unity_Editor || Development_Build` preprocessor directive?